src/Components/CheckSolution/index.tsx

```diff
@@ -0,0 +1,9 @@
+import { FcApproval } from "react-icons/fc";
+
+function CheckSolution(props:{checkSolutionAction: Function}) {
```

Comment:
to be more specific on the function type we can use something like this `() => void`